ads/revjet.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function revjet(global, data) {
+  validateData(data, ['tag', 'key'], ['plc', 'opts', 'params']);
+
+  global._revjetData = Object.assign({}, data);
+
+  loadScript(global, 'https://cdn.revjet.com/~cdn/JS/03/amp.js', undefined, () => {
```

Comment:
1. Can you please mark the `undefined` param with its param name?
2. Shorten line to 80 chars.

```javascript
  loadScript(
      global,
      'https://cdn.revjet.com/~cdn/JS/03/amp.js',
      /* opt_cb */ undefined,
      () => {
        global.context.noContentAvailable();
      });
```